test/new_tests/test_query.py

```diff
@@ -1143,4 +1143,4 @@ def callback(input_tuple):
 
     def test_query_with_base64_cdt_ctx(self):
         bs_b4_cdt = self.as_connection.get_cdtctx_base64({'ctx':ctx_list_index})
```

Comment:
My last comment about this was resolved but I still think this should have its own test file with some more tests. You can see test_get_expression_base64.py as an example. Normally unique methods have their own test files.

Reply:
get_cdtctx_base64 is a support function and there are some tests using that in test_cdtctx.py